src/pow.cpp

```diff
@@ -96,6 +100,14 @@ bool CheckEquihashSolution(const CBlockHeader *pblock, const CChainParams& param
     // H(I||V||...
     crypto_generichash_blake2b_update(&state, (unsigned char*)&ss[0], ss.size());
 
+    #ifdef ENABLE_RUST
```

Comment:
I prefer this indentation while simultaneously disliking precompiler macros like this.